sdk/translation/Azure.AI.Translation.Document/tests/Infrastructure/DocumentTranslationTestEnvironment.cs

```diff
@@ -32,11 +33,13 @@ public DocumentTranslationTestEnvironment()
 
         protected override async ValueTask<bool> IsEnvironmentReadyAsync()
         {
-            string endpoint = Environment.GetEnvironmentVariable(EndpointEnvironmentVariableName);
+            string endpoint = GetOptionalVariable(EndpointEnvironmentVariableName);
             var client = new DocumentTranslationClient(new Uri(endpoint), Credential);
             try
             {
                 await client.GetSupportedDocumentFormatsAsync();
+                await client.GetSupportedGlossaryFormatsAsync();
+                client.GetAllTranslationStatuses().Take(2);
```

Comment:
once filters are implemented, I will use that instead of take

Reply:
I forgot, is this test to make sure that AAD is ready?

Reply:
yes. instead of waiting 5min or any other random period, we have this method that whenever everything works, then it starts executing the tests